mathics/builtin/arithmetic.py

```diff
@@ -999,7 +875,7 @@ class RealNumberQ(Test):
     >> RealNumberQ[0 * I]
      = True
     >> RealNumberQ[0.0 * I]
-     = True
+     = False
```

Comment:
Recall there is no `RealNumberQ` in WMA. 

Eventually I'd like to remove RealNumberQ from Mathics3. It would be okay to have an `eval_RealNumberQ` though. Can we revise this revised to use that?

Reply:
Ok - but please consider doing this _before_ contemplating anything that includes big picture design work.

Reply:
Sure!

Reply:
I was investigating a little bit about this, and I found this discussion:https://mathematica.stackexchange.com/questions/18647/how-to-check-if-an-expression-is-a-real-valued-number

So, in WMA, there is an internal symbol that makes exactly what RealNumberQ does, but with a slightly different name.

Reply:
And actually, in the last version, `RealValuedNumberQ` is a system symbol.